LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2021 Splunk Inc.
+   Copyright 2022 Splunk Inc.
```

Comment:
@rgil-splunk Should it be 2021-2022 or just 2022 is fine?

Reply:
for the sake of consistency we should stick to using the range everywhere if the app had releases in prior years. as for legal purposes I'm not sure if having just 2022 vs the full range carries consequences, would need to double check with legal